plenum/test/node_catchup/test_config_ledger.py

```diff
@@ -0,0 +1,141 @@
+import json
+
+import pytest
+
+from plenum.test.node_catchup.helper import waitNodeDataInequality, \
+    waitNodeDataEquality
+from plenum.test.pool_transactions.helper import \
+    disconnect_node_and_ensure_disconnected, reconnect_node_and_ensure_connected
+
+from plenum.common.util import randomString
+from plenum.test.helper import send_signed_requests, sign_requests, \
+    waitForSufficientRepliesForRequests, check_sufficient_replies_received
+
+from plenum.common.constants import CONFIG_LEDGER_ID, DATA
+from plenum.test.test_config_req_handler import write_conf_op, \
+    TestConfigReqHandler, WRITE_CONF, READ_CONF, read_conf_op
+from plenum.test.test_node import TestNode
+from stp_core.loop.eventually import eventually
+
+
+class NewTestNode(TestNode):
+    def getConfigReqHandler(self):
```

Comment:
Why don't we include `TestConfigReqHandler` to `TestNode`?

Reply:
This config req handler is just use to show that the config ledger can be written to and read from, it is not intended to be used for anything else. Later on when we add some config params (or move from indy-node) we will then have it in the `TestNode`

Reply:
Ok